index.js

```diff
@@ -202,6 +212,14 @@ const checkEmail = () => {
 form.addEventListener('submit', (e) => {
   if (checkEmail() === false) {
     e.preventDefault();
+  } else {
+    const data = {
+      name: namefield.value,
+      email: email.value,
+      text: textarea.value,
+    };
+
+    localStorage.setItem('data', JSON.stringify(data));
   }
 });
```

Comment:
- [ ] Kindly notice that when the user changes the content of any input field in the form the data object is updated and it is saved to the local storage as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=When%20the%20user%20changes%20the%20content%20of%20any%20input%20field%2C%20the%20data%20is%20saved%20to%20the%20local%20storage.).